articles/migrate/concepts-azure-vmware-solution-assessment-calculation.md

```diff
@@ -117,33 +117,33 @@ Here's what's included in an AVS assessment:
+| **Azure Hybrid Benefit** | Specifies whether you have software assurance and are eligible for [Azure Hybrid Benefit](https://azure.microsoft.com/pricing/hybrid-use-benefit/). Although it has no impact on Azure VMware Solution pricing due to the node-based price, customers can still apply the on-premises OS or SQL licenses (Microsoft based) in AVS using Azure Hybrid Benefits. Other software OS vendors will have to provide their own licensing terms, such as RHEL for example. |
 | **vCPU Oversubscription** | Specifies the ratio of number of virtual cores tied to one physical core in the AVS node. The default value in the calculations is 4 vCPU:1 physical core in AVS. API users can set this value as an integer. Note that vCPU Oversubscription > 4:1 may impact workloads depending on their CPU usage. When sizing, we always assume 100% utilization of the cores chosen. |
-| **Memory overcommit factor** | Specifies the ratio of memory over commit on the cluster. A value of 1 represents 100% memory use, 0.5, for example is 50%, and 2 would be using 200% of available memory. You can only add values from 0.5 to 10 up to one decimal place. |
-| **Dedupe and compression factor** | Specifies the anticipated dedupe and compression factor for your workloads. Actual value can be obtained from on-premises vSAN or storage config. These vary by workload. A value of 3 would mean 3x so for 300GB disk only 100GB storage would be used. A value of 1 would mean no dedupe or compression. You can only add values from 1 to 10 up to one decimal place. |
+| **Memory overcommit factor** | Specifies the ratio of memory overcommit on the cluster. A value of 1 represents 100% memory use, 0.5, for example is 50%, and 2 would be using 200% of available memory. You can only add values from 0.5 to 10 up to one decimal place. |
+| **Deduplication and compression factor** | Specifies the anticipated deduplication and compression factor for your workloads. Actual value can be obtained from on-premises vSAN or storage configurations. These vary by workload. A value of 3 would mean 3x so for 300GB disk only 100GB storage would be used. A value of 1 would mean no deduplication or compression. You can only add values from 1 to 10 up to one decimal place. |
 
 ## Azure VMware Solution (AVS) suitability analysis
 
-AVS assessments assess each on-premises VM for its suitability for AVS by reviewing the server properties. It also assigns each assessed server to one of the following suitability categories:
+AVS assessments assess each on-premises vSphere VM for its suitability for AVS by reviewing the server properties. It also assigns each assessed server to one of the following suitability categories:
 
 - **Ready for AVS**: The server can be migrated as-is to Azure (AVS) without any changes. It will start in AVS with full AVS support.
```

Comment:
@rvandenbedem Unless 'AVS' is a UI element that can't be changed. Please search for all other uses of 'AVS' and replace them with 'Azure VMware Solution' where possible. thanks.

Reply:
@suzizuber - all AVS references changed to full name, only 5 "AVS" references are left, since they are tied to the UI.